src/mcp-bundle/tests/DependencyInjection/McpBundleTest.php

```diff
@@ -49,6 +50,10 @@ public function testCustomConfiguration()
 
     public function testMcpLoggerServiceIsCreated()
     {
+        if (!class_exists(MonologBundle::class)) {
+            $this->markTestSkipped('MonologBundle is not installed');
+        }
+
```

Comment:
Since monolog is never installed on the ci, this test will be always skipped, so the rest of the test is useless